File: Python/789. Escape The Ghosts.py
class Solution:
    def escapeGhosts(self, ghosts: List[List[int]], target: List[int]) -> bool:
        dist = abs(target[0]) + abs(target[1])

        for it in ghosts:
            if abs(it[0] - target[0]) + abs(it[1] - target[1]) <= dist:
                return False

        return True

File: C++/789. Escape The Ghosts.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    bool escapeGhosts(vector<vector<int>>& ghosts, vector<int>& target)
    {
        int dist = abs(target[0]) + abs(target[1]);

        for (auto it : ghosts)
        {
            if (abs(it[0] - target[0]) + abs(it[1] - target[1]) <= dist)
                return false;
        }
        return true;

    }
};